src/pages/selfhosted/identity-providers.mdx

```diff
@@ -561,7 +561,23 @@ NETBIRD_IDP_MGMT_EXTRA_USERNAME="Netbird"
 NETBIRD_IDP_MGMT_EXTRA_PASSWORD="<SERVICE_ACCOUNT_PASSWORD>"
 
 ```
-#### Step 5: Continue with the NetBird Self-hosting Guide
+
+#### Step 5: Create a authentication flow for device token authentication
```

Comment:
Let's move Step 5 before the setup.env example. Before the text: "Your authority OIDC configuration will be available under:"